.eslintrc.js

```diff
@@ -483,7 +483,7 @@ const rules = {
   '@typescript-eslint/consistent-indexed-object-style': 1,
   '@typescript-eslint/consistent-type-assertions': [2, { assertionStyle: 'as', objectLiteralTypeAssertions: 'never' }],
   '@typescript-eslint/consistent-type-definitions': [1, 'interface'],
-  '@typescript-eslint/consistent-type-imports': [1, { prefer: 'no-type-imports' }],
+  '@typescript-eslint/consistent-type-imports': 1,
```

Comment:
I've tweaked this so we'll now use `type` imports, this is to help babel elide type imports, otherwise it cannot know to remove some imports, especially in files like `src/index.ts`.